From fa8a70da8d62ddafa8504b84cb7256248fcd7627 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Steven!=20Ragnar=C3=B6k?= <steven@nuclearsandwich.com>
Date: Tue, 12 Sep 2017 11:29:54 -0400
Subject: [PATCH 2/2] Fix dependency template.

---
 debian/control.em | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/debian/control.em b/debian/control.em
index 38695d5..1065a3a 100644
--- a/debian/control.em
+++ b/debian/control.em
@@ -8,7 +8,7 @@ Standards-Version: 3.9.2
 
 Package: @(Package)
 Architecture: any
-Depends: @(DebhelperDepends), ${misc:Depends}, ros-kinetic-kobuki-driver, @(', '.join(Depends))
+Depends: ${shlibs:Depends}, ${misc:Depends}, ros-kinetic-kobuki-driver, @(', '.join(Depends))
 @[if Conflicts]Conflicts: @(', '.join(Conflicts))@\n@[end if]@
 @[if Replaces]Replaces: @(', '.join(Replaces))@\n@[end if]@
 Description: @(Description)
-- 
2.7.4

